kernel/panic_entry/src/lib.rs

```diff
@@ -132,3 +132,16 @@ fn oom(_layout: core::alloc::Layout) -> ! {
     error!("\n(oom) Out of Heap Memory! requested allocation: {:?}", _layout);
     panic!("\n(oom) Out of Heap Memory! requested allocation: {:?}", _layout);
 }
+
+// TODO: Are these truly unreachable?
```

Comment:
these two function impls are fine for now, just replace this comment with one above each of these functions that explains how the build of std-reliant apps was erroring out without these present.